docs/changehistory/NextVersion.md

```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
     alert("Integrated graphics are in use. If a discrete GPU is available, consider switching your device or browser to use it.");
 ```
 
+### Fixed inconsistent property representation in a property grid
+
+Previously when using `nestedRelatedProperties` attribute, nested related properties were merged or not merged depending on whether at least one property of an intermediate class was selected. Now in both cases properties are not merged.
+
+```json
+{
+  "id": "example",
+  "rules": [
+    {
+      "ruleType": "Content",
+      "specifications": [
+        {
+          "specType": "ContentInstancesOfSpecificClasses",
+          "classes": {
+            "schemaName": "BisCore",
+            "classNames": ["GeometricModel3d"],
+            "arePolymorphic": true
+          }
+        }
+      ]
+    },
+    {
+      "ruleType": "ContentModifier",
+      "class": {
+        "schemaName": "BisCore",
+        "className": "Model"
+      },
+      "relatedProperties": [
+        {
+          "propertiesSource": {
+            "relationship": {
+              "schemaName": "BisCore",
+              "className": "ModelContainsElements"
+            },
+            "direction": "Forward",
+            "targetClass": {
+              "schemaName": "Generic",
+              "className": "PhysicalObject"
+            }
+          },
+          "properties": "_none_",
+          "nestedRelatedProperties": [
+            {
+              "propertiesSource": {
+                "relationship": {
+                  "schemaName": "BisCore",
+                  "className": "GeometricElement3dIsInCategory"
+                },
+                "direction": "Forward",
+                "targetClass": {
+                  "schemaName": "BisCore",
+                  "className": "Category"
+                }
+              },
+              "properties": "*",
+              "handleTargetClassPolymorphically": true
+            }
+          ],
+          "handleTargetClassPolymorphically": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+|                               | before                                                                                                                            | after                                                                                                                             |
```

Comment:
Let's add this to the first column header: **Value of `"properties"` attribute**.

And then change values for the column to `"_none_"` and `["UserLabel"]`.